### PR TITLE
WIP: Crazy Cataclysm Changes for Bright Nights

### PR DESCRIPTION
**Pull request by Pericles1848**
#### Summary

[Crazy Cataclysm]: Changes to crazy cataclysm and a plan to maintain it for BN.

#### Purpose of change

Crazy Cataclysm is currently not maintained in BN and is littered with content that, to be quite frank, isn't interesting or funny. This aims to change it as well as hopefully encourage others to add their own ideas.

#### Describe the solution

First step is changing fire melees to make them more interesting and powerful as weapons. The next steps include removing the loose collection of unrelated memes and establish an uniform vision/balance for the mod. More content will be added after that point.

#### Describe alternatives you've considered

Make another "reference" mod or similar "crazy" mod, not change CrazyCata. 

#### Testing

WIP

#### Additional context

None atm

### CODE REVIEW

File: data/mods/CrazyCataclysm/crazy_monsters.json [hunk 1]
```diff
@@ -14,65 +14,6 @@
...
-    "melee_dice": 3,
-    "melee_dice_sides": 6,
-    "melee_cut": 4,
-    "dodge": 2,
-    "armor_cut": 15,
-    "armor_stab": 15,
-    "armor_acid": 3,
-    "vision_day": 30,
-    "vision_night": 3,
-    "death_drops": "skeltal_drops",
```
Comment (Coolthulhu) [inline data/mods/CrazyCataclysm/crazy_monsters.json:43]: This item group is only referenced here.

Comment (Pericles1848) [reply data/mods/CrazyCataclysm/crazy_monsters.json:43]: looks like I missed that, sorry

**Comment by Coolthulhu** [comment]
If this is ready, you should mark it as ready for review.

**Comment by Coolthulhu** [comment]
Is this being worked on, ready or stale?

**Comment by DrPariah** [comment]
I was going to suggest that in Crazy Bright Nights we change the names of all fake and inferior melee weapons to MALL NINJA(TM) brand weapons. The inferior ones are MALL NINJA's "Battle Ready" line, which are made of slightly better grade pot metal.

**Comment by Coolthulhu** [comment]
Closing as stale

**Pull request closed without merging**